components/fxa-client/src/http_client.rs

```diff
@@ -68,6 +74,12 @@ pub trait FxAClient {
         profile_access_token: &str,
         etag: Option<String>,
     ) -> Result<Option<ResponseAndETag<ProfileResponse>>>;
+    fn ecosystem_anon_id(
```

Comment:
API nit: this feels like it should contain a verb, like `set_ecosystem_anon_id` or similar; at first read I thought this was going to *return* the anon_id, which it doesn't.